driver-core/src/test/java/com/datastax/driver/core/SpeculativeExecutionTest.java

```diff
@@ -100,14 +102,14 @@ public void should_not_start_speculative_execution_if_first_execution_retries_bu
         // will retry once on this node:
         scassandras.node(1).primingClient().prime(PrimingRequest.queryBuilder()
                         .withQuery("mock query")
-                        .withConsistency(PrimingRequest.Consistency.TWO)
-                        .withResult(PrimingRequest.Result.read_request_timeout)
+                .withConsistency(Consistency.TWO)
+                .withResult(Result.read_request_timeout)
```

Comment:
Weird, even with save actions turned on it still doesn't format it.  Must be some edge case, in any case i'll fix it.